utils/pre-swap-checker/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@airswap/pre-swap-checker",
-  "version": "0.0.3",
+  "version": "0.1.3",
```

Comment:
need to be marked as beta

Reply:
even though it's not being deployed to npm?

Reply:
Yeah that is what was concluded after a discussion on slack. We beta them anyway.